src/fsharp/xlf/FSComp.txt.es.xlf

```diff
@@ -109,7 +109,7 @@
       </trans-unit>
       <trans-unit id="followingPatternMatchClauseHasWrongType">
         <source>All branches of a pattern match expression must return values of the same type as the first branch, which here is '{0}'. This branch returns a value of type '{1}'.</source>
-        <target state="translated">Todas las ramas de una expresión de coincidencia de patrón deben devolver valores del mismo tipo. La primera rama devolvió un valor de tipo "{0}", pero esta rama devolvió un valor de tipo "\{1 \}".</target>
+        <target state="new">All branches of a pattern match expression must return values of the same type as the first branch, which here is '{0}'. This branch returns a value of type '{1}'.</target>
```

Comment:
🤔?

Reply:
These get machine-translated into target languages, so no need to worry about this being modified.

Reply:
Thanks @cartermp . I'm just wondering why it turns back to English.

Reply:
Just a part of the build.